### PR TITLE
docs(ios): updates iOS app help for 17.0 banner changes

### LINKED ISSUE #10630: docs(ios): update banner documentation
We need to refresh any references to the banner and document the usage of it with the changes in v17.

See also #10629.

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #10630.

I used the Markdown changes from #10691 - the Android version of this update - as the baseline, then went along and updated all relevant images I found.

For two images, I did swap which keyboard was being used:
- In one, I had the displayed keyboard match the keyboard displayed in Android help for the same thing
- In the other, I used the keyboard originally used in the image for the prior point, as the original image _here_ was used in the _new_ image for the prior point.

While I was at it, I decided to throw in one extra freebie - an animated gif demonstrating down-flicks, as no such gif was added in #10763.

I also deleted an image or two that appears to be unused - one of which was _definitely_ outdated.

## User Testing

TEST_IOS_HELP:  Navigate throughout the iOS app help and report on any broken links and/or missing images.

### CODE REVIEW

File: ios/help/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ title: Keyman for iPhone and iPad 17.0 Help
 
 
 ## [Using Keyman for iPhone and iPad](basic/)
+* [Using the Banner on the Keyboard](using-the-banner)
```
Comment (darcywong00) [inline ios/help/index.md:18]:
```suggestion
* [Using the Banner on the Keyboard](basic/using-the-banner)
```

**Comment by jahorton** [comment]
Yep, my old frenemy copy-and-paste struck again; forgot to adjust the path on the top-level.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by jahorton** [comment]
TEST_IOS_HELP: PASSED

I downloaded the TestFlight build and can verify that the previously-broken link has been properly fixed.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.306-beta](https://keyman.com/downloads/releases/beta/17.0.306)

**Pull request merged**